code/modules/clothing/spacesuits/rig/suits/light.dm

```diff
@@ -112,7 +112,7 @@
 	name = "stealth suit control module"
 	suit_type = "stealth"
 	desc = "A highly advanced and expensive suit designed for covert operations."
-	icon_state = "stealth_rig"
+	icon_state = "ninja_rig" //supposed to be "stealth_rig", but as it currently only has a semi-copied ninja rig sprite, we can just use them directly.
```

Comment:
Well, while you use the stealth rig, the helmet did just remove your hair, and that's it, since there is a missing sprite. And instead of copy-pasting the ninja rig sprites, which the existing sprites already are, i decided to just use the ninja rig sprites.